x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -49,8 +56,17 @@ import { ruleStatusServiceFactory } from './rule_status_service';
 import { buildRuleMessageFactory } from './rule_messages';
 import { ruleStatusSavedObjectsClientFactory } from './rule_status_saved_objects_client';
 import { getNotificationResultsLink } from '../notifications/utils';
+import { buildEqlSearchRequest } from '../../../../common/detection_engine/get_query_filter';
+import { bulkInsertSignals } from './single_bulk_create';
+import { buildSignalFromSequence, buildSignalFromEvent } from './build_bulk_body';
+import { buildParent } from './build_signal';
 import { createThreatSignals } from './threat_mapping/create_threat_signals';
 
+interface Sequence {
+  buildingBlocks: BaseSignalHit[];
+  sequenceSignal: BaseSignalHit;
+}
```

Comment:
I'd put this interface into the `type.ts` and export it maybe. Might keep this file's size low as we know it's kind of large right now.